scripts/rollup/bundles.js

```diff
@@ -45,7 +45,12 @@ const bundles = [
       sourceMap: false,
     },
     entry: 'src/isomorphic/React.js',
-    externals: [],
+    externals: [
+      'create-react-class/factory',
+      'prop-types',
+      'prop-types/checkPropTypes',
```

Comment:
Does only isomorphic use checkPropTypes like this? I guess I would have thought that context type checking does too.